### PR TITLE
[Cart] Minor fixes

### PR DESCRIPTION
**Pull request by lchrusciel**

### CODE REVIEW

File: src/Query/Carts.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ public function __construct(
         $this->cartViewFactory = $cartViewFactory;
     }
 
-    public function findByToken(?string $orderToken): CartSummaryView
+    public function findByToken(string $orderToken): CartSummaryView
```
Comment (lchrusciel) [inline src/Query/Carts.php]: `findByToken` -> `findOneByToken`

File: src/Query/Carts.php [hunk 2]
```diff
@@ -10,7 +10,7 @@
 use Sylius\ShopApiPlugin\View\CartSummaryView;
 use Webmozart\Assert\Assert;
 
-final class CartQuery implements CartQueryInterface
+final class Carts implements CartsInterface
```
Comment (pjedrzejewski) [inline src/Query/Carts.php]: What is the rationale behind such naming? (Not saying it is wrong, I just want to understand.)

Comment (lchrusciel) [inline src/Query/Carts.php]: I'm just trying to find a best one. From the DDD perspective, we should have most meaningful words for classes. But `Carts` doesn't convince me either. I will probably move back to CartQuery, but PR has some other fixes, so it still makes sense ;)

Comment (pamil) [inline src/Query/Carts.php]: From my POV they're nothing else but repositories, query does not find anything on its own.

File: src/Query/CartQuery.php [hunk 3]
```diff
@@ -34,7 +34,7 @@ public function __construct(
         $this->cartViewFactory = $cartViewFactory;
     }
 
-    public function findByToken(?string $orderToken): CartSummaryView
+    public function findOneByToken(string $orderToken): CartSummaryView
```
Comment (pamil) [inline src/Query/CartQuery.php]: What about keeping to the following convention?
 - `findSomething` if method returns `Something` or `null`
 - `getSomething` if method always returns `Something`

**Comment by pamil** [comment]
The comment I made might get unnoticed so I'm linking to it here -
 (https://github.com/Sylius/SyliusShopApiPlugin/pull/178#discussion_r125939358).

**Comment by pamil** [comment]
Thanks @lchrusciel! 🎉

**Pull request merged**